mm2src/coins/lp_coins.rs

```diff
@@ -287,7 +288,11 @@ use utxo::{BlockchainNetwork, GenerateTxError, UtxoFeeDetails, UtxoTx};
 pub mod nft;
 use nft::nft_errors::GetNftInfoError;
 
+pub(crate) mod nft_storage;
```

Comment:
Can we stack the `mod` statements together? It will be good to see all of them in a single place.

Reply:
right now we have only this for nft in lp_coins.rs
```rust
pub mod nft;
use nft::nft_errors::GetNftInfoError;
```